content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -64,11 +64,7 @@ These values populate the `config` section in the `Pulumi.<stack>.yaml` file, sp
 
 ### Environment
 
-If you've configured [environments](/docs/pulumi-cloud/esc) with your organization, you can specify one to use with the resulting stack. Environments enable teams to manage hierarchical collections of configuration and secrets without needing to re-specify all of them during project creation.
-
-{{% notes type="info" %}}
-Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end and can be set on each stack. However, by creating an environment with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
-{{% /notes %}}
+If you've configured [environments](/docs/pulumi-cloud/esc) with your organization, you can specify one to use with the resulting stack. Environments enable teams to define and share collections of configuration and secrets across multiple Pulumi projects and stacks, such as your `AWS_SECRET_ACCESS_KEY` which will be needed to configure your deployment settings. 
 
 ### Repository
```

Comment:
Just a note to ask about provisioning private repos (as folks are likely to wonder this) and/or what to do if we only support creating public repos.

Reply:
all add this to my docs feedback notes; then come back and clarify when we update them.